server/basicServices/basicServices/BasicServicesService.js

```diff
@@ -37,7 +37,8 @@ const createHttpError = require('http-errors');
 const HttpServerInterface = require('onf-core-model-ap/applicationPattern/onfModel/models/layerProtocols/HttpServerInterface');
 const OperationClientInterface = require('onf-core-model-ap/applicationPattern/onfModel/models/layerProtocols/OperationClientInterface');
 
-const integerProfile = require('onf-core-model-ap/applicationPattern/onfModel/models/profile/IntegerProfile'); 
+const integerProfile = require('onf-core-model-ap/applicationPattern/onfModel/models/profile/IntegerProfile');
+const HttpClientInterface = require('onf-core-model-ap/applicationPattern/onfModel/models/layerProtocols/HttpClientInterface');
```

Comment:
I can see a duplicity in import of httpClientInterface module. Can we remove this and update corresponding lines of HttpClientInterface to httpClientInterface.

Reply:
Thanks, removed the duplicate entry as mentioned.